### ISSUE TITLE
Question about vibration and shock sensitivity

### ISSUE THREAD

**Issue by unipoco** [stefanoaz/Particle-counter-V2#1]
Thank you for your efforts on this project. I built a particle detector from scratch and it worked beyond my expectations. I noticed something odd about the PIN diode sensors, they seemed to be very sensitive to vibration. When I tap the table near the sensor, I get a false signal on the oscilloscope. 

I have heard that commercial dosimeter products use vibration sensors to compensate for this error, is this the only way to do this or are there other ways to improve resistance to vibration interference?

(Ps: I used slightly different materials, i.e. 1.6mm PCB boards, and the op-amps were OPA2607 and TLV9062 respectively. theoretically these would not be affected by vibration, but it's better to state it so that it's easier to control the variables.)

**Comment by stefanoaz** [comment]
Hello. This is an amplifier with a very high impedance input - that's why it has a differential path in the front end - to subtract out any interfering signals which would arrive at the same polarity (unlike the diode signal). But this subtraction is not perfect. You should see if flipping a light switch in the room also produces a false signal.

One good idea to try if you see a signal from the light switch test, is to use adhesive copper tape to make a shield around the diodes. Ground this shield with solder, to the PCB ground plane at one point only. Make sure the copper tape encloses the diodes from both sides of the board, but it doesn't have to be light tight as long as you already have a light barrier.

That would cover EMI interference. If you are really seeing something piezoelectric, I don't think the diodes and opamps are intrinsically noisy in that way. HOWEVER - high-K ceramic capacitors with dielectrics like X7R and X5R - are notorious for having piezoelectric noise when vibrated. There should only be NPO (C0G) dielectric caps in the front-end signal path. The X7R and X5R are fine for bypass and the charge pump voltage multiplier. This is specified in the BOM.

Hope that helps.

**Comment by unipoco** [comment]
Thanks a lot for taking the time to review this. I had already used grounded copper tape because it is both light-tight and perfect shielding material. The EMI immunity of the unit is quite impressive and I only found that high power walkie-talkies (which is literally EMI bombardment) affected it in my tests. 

Your mention of the X7R capacitor issue gave me a very important tip! I couldn't find 15nF C0G capacitors, so I used 15nF X7R for C4, C5, C10 and C11. Now it looks like a huge mistake since vibration on these capacitors would send a noise signal directly into the front-end amplifier.

I'll replace them with C0G capacitors to redo the test. Thanks again for pointing this out, saved me a lot of time troubleshooting and taught me something new as well!
